packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -170,13 +163,13 @@ const ApplicationDetails = () => {
     </>
   );
 
-  const AdvancedSettingsPage = oidcConfig && (
+  const AdvancedSettingsPage = data && (
     <>
       <FormField title="admin_console.application_details.token_endpoint">
-        <CopyToClipboard className={styles.textField} value={oidcConfig.token_endpoint} />
+        <CopyToClipboard className={styles.textField} value={data.oidcConfig.tokenEndpoint} />
```

Comment:
should we change the default oidcConfig key case behavior?

Reply:
per offline discussion, keep camel case in our logto-core APIs.